### PR TITLE
add jhohannes profile

### PR DESCRIPTION
**Pull request by JhohannesK**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1283"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the `package-lock.json` file. Let us know if you need any help. :)

**Comment by JhohannesK** [comment]
> Please revert the changes in the `package-lock.json` file. Let us know if you need any help. :)

how do I that?

**Review by ahmed82 — APPROVED** [review]
Looks Ok.

**Review by ahmed82 — CHANGES_REQUESTED** [review]
You should make the change < adding new file only that contain your name>, not the 'package-lock.json'

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.59.1",
+      "version": "0.64.1",
       "license": "MIT",
```
Comment (ahmed82) [inline package-lock.json:10]: The change is the version that you need to revert.

**Comment by JhohannesK** [comment]
I have reverted the file please

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**